package-lock.json

```diff
@@ -110,7 +110,7 @@
         "@nrwl/linter": "15.3.0",
         "@nrwl/nest": "15.3.3",
         "@nrwl/node": "15.3.3",
-        "@nrwl/nx-cloud": "latest",
+        "@nrwl/nx-cloud": "*",
```

Comment:
@itrulia is that ok?

Reply:
Isn't * all tags? If yes, that could be a problem but shouldn't matter for nx-cloud.